lambda-deployment-orca/src/main/java/com/amazon/aws/spinnaker/plugin/lambda/eventconfig/LambdaUpdateEventConfigurationTask.java

```diff
@@ -61,11 +57,10 @@
     @Override
     public TaskResult execute(@NotNull StageExecution stage) {
         logger.debug("Executing LambdaUpdateEventConfigurationTask");
-        cloudDriverUrl = props.getCloudDriverBaseUrl();
```

Comment:
this shouldn't be removed. breaks update configurations. instead of removing, you will want to move it closer to L174 where cloudDriverUrl is actually used.

Reply:
*shake* dang it... my bad sorry